### PR TITLE
dynamic copyright, timecop gem. closes #569

### LINKED ISSUE #569: Update Copyright in footer
Still living in 2016... !

Update dynamically.

### PR DESCRIPTION
**Pull request by sharpattack**

### CODE REVIEW

File: spec/views/_footer.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      
+      it 'shows a year later' do
+        Timecop.freeze(next_yr) do
+          render
+          expect(rendered).to have_text("© #{next_yr.year} Northwestern University")
+        end
+      end
+    end
+  end
+end
```
Comment (phebal) [inline spec/views/_footer.html.erb_spec.rb]: Needs and EOF :) I like that you're using view specs instead of capybara, it's much lighter.

Comment (sharpattack) [reply spec/views/_footer.html.erb_spec.rb]: you mean for timecop?

**Comment by sharpattack** [comment]
will rebase after you merge your other two PR's @phebal

File: spec/views/_footer.html.erb_spec.rb (hunk 1 above)
Comment (phebal) [reply spec/views/_footer.html.erb_spec.rb]: I mean for the file (`_footer.html.erb_spec.rb`).  See this link:
https://github.com/galterlibrary/digital-repository/pull/581/files/16d589ea1956032b34fa9d9ab7cb155d31764027#diff-ba8e672e8cb04fe32f6e7ea2e88b214e
The little `no new line at the end of file` badge. Just being my anal self.

Comment (sharpattack) [reply spec/views/_footer.html.erb_spec.rb]: ahhh i see, haha. i try to keep that in mind. i forget too often it seems ha.

**Pull request merged**